src/pc/mod.rs

```diff
@@ -61,6 +67,26 @@ pub(crate) enum ParachainCommanderMode {
 	Prometheus(ParachainCommanderPrometheusOptions),
 }
 
+#[derive(Clone, Debug, strum::Display)]
+enum ParachainCommanderParaId {
```

Comment:
If we make it an Option then it will be treated as a default when no para-id is specified. I don't think that tracking all parachains is a sane default for an end user. So I'd argue that `para-id=all` is a cleaner UX from this point of view.

Reply:
I see what you mean, the CLI would basically be a wall of fast scrolling text if no para id is specified.

Reply:
I still feel this is a bit clunky ... Could we instead have separate bool parameter ` --all` and ParaId would just be a Vec as before ?

Reply:
Yes, this is also an option.